src/main/java/com/iota/iri/controllers/AddressViewModel.java

```diff
@@ -38,6 +42,30 @@ private AddressViewModel(Address hashes, Indexable hash) {
         this.hash = hash;
     }
 
+    /**
+     * Constructor for an {@link Address} set controller from an existing {@link Address} set.
+     * The resulting hash list is sorted
+     *
+     * @param tangle The tangle reference for the database to find the {@link Address} set in
+     * @param hash The hash identifier for the {@link Address} set that needs to be found
+     * @return The {@link AddressViewModel} controller generated
+     * @throws Exception Thrown if the database cannot load an {@link Address} set from the reference {@link Hash}
+     */
+    public static List<TransactionViewModel> loadAsSortedList(Tangle tangle, Indexable hash) throws Exception {
+        Address hashes = (Address) tangle.load(Address.class, hash);
+        return hashes.set.stream().map(item -> {
+            try {
+                if (tangle.exists(Transaction.class, item)) {
+                    return TransactionViewModel.fromHash(tangle, item);
+                }
+                return null;
```

Comment:
why return null? Why not do a `.filter(item ->  (tangle.exists(Transaction.class, item))` so you won't have nulls in your list